crates/cxx-qt-gen/src/parser/property.rs

```diff
@@ -59,18 +64,26 @@ pub struct ParsedQProperty {
 fn parse_meta_name_value(name_value: &MetaNameValue) -> Result<(Ident, Ident)> {
     let ident = name_value.path.require_ident()?.clone();
     let expr = &name_value.value;
-    let func_signature: Ident;
 
-    if let Expr::Path(path_expr) = expr {
-        func_signature = path_expr.path.require_ident()?.clone();
+    // Only cxx_name and rust_name can use string literals for values
+    let value = if ident == "cxx_name" || ident == "rust_name" {
+        let string = expr_to_string(expr).map_err(|_| {
+            Error::new(
+                ident.span(),
+                "cxx_name and rust_name must use string values like `cxx_name = \"myName\"`!",
+            )
+        })?;
+        format_ident!("{}", string)
```

Comment:
Do not use format_ident here, as it will panic if the string cannot be converted to an ident.
So if I did e.g. `rust_name = "hello world"`, cxx-qt-gen would panic, instead of handling this gracefully.

If you can get hold of a LitStr instance, you should be able to call `parse::<Ident>` on it:
https://docs.rs/syn/latest/syn/struct.LitStr.html#method.parse

Reply:
Done using syn::parse_string